extensions/wikidata/src/org/openrefine/wikidata/qa/WikidataConstraintFetcher.java

```diff
@@ -94,7 +92,10 @@
     public static String ALLOWED_ENTITY_TYPES_QID = "Q52004125";
     public static String ALLOWED_ITEM_TYPE_QID = "Q29934200";
     public static String ALLOWED_ENTITY_TYPES_PID = "P2305";
-    
+
+    public static String CONFLICTS_WITH_CONSTRAINT_QID = "Q21502838";
+    public static String CONFLICTS_WITH_PROPERTY_PID = "P2306";
+    public static String ITEM_OF_PROPERTY_CONSTRAINT_PID = "P2305";
```

Comment:
This appears to duplicate `ALLOWED_ENTITY_TYPES_PID` just above.

Reply:
While these happen to have the same value for Wikidata, it might not be true of all Wikibase instances (these constants should become user-configurable once we add proper Wikibase support). That being said the name of the constant could probably be adapted to reflect that it is specifically for that constraint.